packages/ra-ui-materialui/src/input/AutocompleteInput.spec.tsx

```diff
@@ -1119,7 +1119,7 @@ describe('<AutocompleteInput />', () => {
                 <SimpleForm
                     mode="onBlur"
                     onSubmit={jest.fn()}
-                    defaultValues={{ role: 2 }}
+                    defaultValues={{ language: 'ang' }}
```

Comment:
To me these tests have nothing to do with changing the `defaultValues` (I think it's a copy paste error), hence I changed that to keep the same `defaultValues` across rerenders.